src/pages/publishers.astro

```diff
@@ -0,0 +1,69 @@
+---
+import { getLangFromUrl, useTranslations } from '../i18n/utils'
+import Base from '../layouts/Base.astro'
+import {
+  heroContent,
+  publisherTags,
+  toolsContent,
+  walletConnectionContent,
+  wordPressPluginContent,
+} from '../data/publishers.ts'
+import AudienceHero from '../components/pages/AudienceHero.astro'
+import HowItWorks from '../components/pages/HowItWorks.astro'
+import SplitLayout from '../components/pages/SplitLayout.astro'
+
+import { HowItWorksPublishers } from '../data/publishers.ts'
+
+const lang = getLangFromUrl(Astro.url)
+const t = useTranslations(lang)
+---
+
+<Base
+  title={t('pages.publishers.title')}
+  description={t('pages.publishers.description')}
+>
+  <AudienceHero content={heroContent} pills={publisherTags}>
+    <p>
+      Earn money every second someone spends on your content. Our
+      pay-as-you-browse model turns attention into direct, automatic payments.
+      Web Monetization is fast, secure, private, and integrates seamlessly with
+      your current setup.
+    </p>
+    <p>All it takes is a wallet address and a single line of code.</p>
+  </AudienceHero>
+  <HowItWorks content={HowItWorksPublishers}>
```

Comment:
We might want to vertical center this image on smaller screens:
<img width="844" height="1419" alt="image" src="https://github.com/user-attachments/assets/53e08848-97ad-4d0c-b074-20a05bf0f069" />